src/app/styles/terminal.sass

```diff
@@ -1,13 +1,184 @@
+@import '../../../node_modules/xterm/src/xterm.css'
+
 .window-terminal-container
   position: relative
-  display: block
-  color: #ffffff
+  display: inline-block
+  color: #ffffff !important
   background: #000000
-  height: 600px
-  -webkit-font-smoothing: auto
+  height: 700px
+  width: 100%
+  font-family: Monaco, monospace
+  font-size: 12px
+
```

Comment:
add `padding: 10px` here please, it looks better.

Reply:
`padding: 5px 10px;` looks good.

Reply:
done